docs/Installation.md

```diff
@@ -94,8 +94,8 @@ In that case, before you run the test, make sure you have created
   * also make sure that the docker config, that is encoded in the value of `QUAY_TOKEN` environment variable, contains a correct credentials required to push to `test-images` repo. And make sure the robot account or user account has the **write** permissions set for `test-images` repo which is required by the tests to push the generated artifacts.
 * fork following GitHub repositories to your org (specified in `MY_GITHUB_ORG` env var)
   * https://github.com/redhat-appstudio-qe/devfile-sample-hello-world (for running build-service tests)
-  * https://github.com/redhat-appstudio-qe/hacbs-test-project (for rhtap-demo test)
-  * https://github.com/redhat-appstudio-qe/strategy-configs (for rhtap-demo test)
+  * https://github.com/redhat-appstudio-qe/hacbs-test-project (for konflux-demo test)
+  * https://github.com/redhat-appstudio-qe/strategy-configs (for konflux-demo test)
```

Comment:
Wouldn't it be confusing if we start referring rhtap-demo test suite as konflux-demo when we have just renamed the directory and not the actual test?

Reply:
you are tight, 
it was my miss, I've renamed all and missed the test itself , thanks for the catch